server/src/main/java/io/druid/server/coordination/DruidServerMetadata.java

```diff
@@ -40,7 +43,10 @@ public DruidServerMetadata(
       @JsonProperty("maxSize") long maxSize,
       @JsonProperty("type") String type,
       @JsonProperty("tier") String tier,
-      @JsonProperty("priority") int priority
+      @JsonProperty("priority") int priority,
+      @JsonProperty("service") String service,
+      @JsonProperty("hostText") String hostText,
```

Comment:
Can you clarify the difference between `hostText`, `host`, `type`, `tier`, `name`, and `service`?